workshop/exercise-1/1-questions.md

```diff
@@ -1,98 +1,101 @@
 # HTML Comprehension Questions
 
+this is a test
 
 ## Q1 - For each of the following HTML documents, is the HTML valid?
 
 Type true/false in the provided [ ].
 
-a) [ ] `<div><span>hello</div></span>`
+a) [false] `<div><span>hello</div></span>`
```

Comment:
true, this is valid html, this will print `hello` to the screen

Reply:
backtick hello backtick, is what it will print (sorry cant figure out how to write backtick here wiyhout starting code block)